clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -778,7 +658,7 @@ def p4ctl_add_entry(client, bridge, tbl_name, flow):
     """
     add-entry SWITCH TABLE MATCH_KEY ACTION ACTION_DATA
     Example:
-        ovs-p4ctl add-entry br0 pipe.filter_tbl
+        p4rt-ctl add-entry br0 pipe.filter_tbl
```

Comment:
Since this is a sample command line, you might consider specifying a backslash at the end of the first line, since that's what you would have to do if you entered this into bash ...

... always assuming that specifying a single backslash at the end of a line within triple quotes in Python does not have some obnoxious side-effect.